Task Estimation Draft - Initial Contribution

FTouray opened the pull request:
This PR introduces the draft version of the Task Estimation in Scrum section. 

It includes:
Overview & Importance of task estimation
Key Guidelines for Effective Estimation
Different Types of Estimation (Planning Poker, T-Shirt Sizing, Affinity Estimation, etc.)
Common Mistakes & Bad Practices

@Jeremy2408  will complete the remaining sections.
Additional diagrams and refinements may be added based on feedback.

**Note: Image might not display until branch has been merged.**

jaynwanze reviewed (approved):
I am very happy with the improvement in quality of content provided compared to the initial draft. The mix of photos and concise clear key points where relevant make it perfect for a easy to read report!

jaynwanze reviewed (changes requested):
An additional diagram or image will suffice

FTouray commented:
> An additional diagram or image will suffice

@jaynwanze  Thanks for your suggestions! I have:
- Added a diagram in Section 3.
- Added a link to a YouTube video that explains common mistakes(Section 5).
- Fixed formatting issues.

OreAdebiyi14 reviewed (approved):
The updates looks good and the new links, and diagrams explain what task estimation is in detail.

Jeremy2408 reviewed (approved):
This is a great draft that I can pick up from

jaynwanze reviewed (approved):
Excellent, all good from my side!
